swift/Sources/SignalClient/Aes256Ctr32.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright 2020 Signal Messenger, LLC
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+import SignalFfi
+import Foundation
+
+public class Aes256Ctr32: ClonableHandleOwner {
+    public init<KeyBytes, NonceBytes>(
+        _ key: KeyBytes,
+        _ nonce: NonceBytes,
+        _ initial_ctr: UInt32) throws
```

Comment:
Nitpick: even though there are Rust naming conventions throughout much of the code (especially the tests), we've tried to keep the public interface to Swift naming conventions (camelCase).

Reply:
I copied the naming pattern from `Aes256GcmSiv` but happy to update :)

Reply:
\me shakes fist at Jack, then at past self